plugins/modules/aci_cloud_aws_provider.py

```diff
@@ -66,7 +66,7 @@
     password: SomeSecretePassword
     tenant: ansible_test
     account_id: 111111111111
-    is_trusted: yes
+    is_trusted: true
```

Comment:
These "is_trusted" and "is_account_in_org" should also be changed in the test file, but the test file is missing in PR.

Reply:
Please also check the aci_cloud_aws_provider test file for is_trusted: no, because i spot few usages of it

Reply:
resolved